spectrum-node/Cargo.toml

```
@@ -2,10 +2,11 @@
 name = "spectrum-node"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.64.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-libp2p = "0.43.0"
+libp2p = "0.49.0"
```

```suggestion
libp2p = {version = "0.49.0", features = ["ping", "tcp", "dns", "async-std", "websocket", "noise", "mplex", "yamux"]}
```
To address compilation errors:
 - could not find `ping` in `libp2p` and
 - cannot find function `development_transport` in crate `libp2p`

brought about by version `0.49`.

---

My bad, forgot to update spectrum-node module.